chapters/en/slides/module5/module5_05.Rmd

```diff
@@ -0,0 +1,393 @@
+---
+params:
+  dynamictitle: "module5_05"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+
+# Effective use of color for quantitative data
+
+Notes:
+This slide deck describes how to choose color to accurately represent numerical data.
+
+---
+
+## Repetition of hue, lightness, and saturation
+
+<center><img src="/module5/hsl-questions.png" width="50%"></img></center>
+
+Notes:
+In the last slide deck we varied hues to represent categorical values,
+with a focus on the distinction between color hues/categories.
+When choosing a color scheme to represent quantitative data,
+we will instead focus on sequential, perceptually uniform changes in lightness.
+
+Sequential changes in lightness ensures that uniform changes in the values of the data
+are perceived as uniform changes in the color space.
+In other words,
+we want it to be intuitive 
+how the colors relate the numerical values they represent,
+so that we can accurately understand how the data values vary
+when looking at the changing colors.
+
+---
+
+## Lightness is more effective than hue to communicate changes in quantitative data
+
+```{python}
+# TODO hide code
+import altair as alt
+from vega_datasets import data
+
+cars = data.cars()
+chart = alt.Chart(cars).mark_point(size=50, filled=True).encode(
+    x='Horsepower',
+    y='Miles_per_Gallon',
+    color=alt.Color('Origin', title=None))
+((chart.encode(alt.Color('Horsepower', scale=alt.Scale(scheme='tableau10')))
+| chart.encode(alt.Color('Horsepower')))
+ .resolve_scale(color='independent'))
+```
+
+Notes:
+In the plots on this slide,
+we have encoded the same variable (Horsepower) to be represented by both the x-axis and the color.
+This is rarely done in practice,
+but it is useful to contrast and understand color schemes
+because we know that the colors of the points should change in a perceptually uniform manner
+as we move from left to right along the x-axis.
```

Comment:
I am not a big fan of this, but I can see where you are coming from pedagogically. So if you want to stick with this, I suport, however, I think you need to add some slides at the end of this deck where you encode a 3rd quantitative variable so that there is no redundancy, and talk about how to correctly interpret it, and the challenges of doing so.

Reply:
This is the only way I can think of where once can actually asses how the color schemes relate to the data, since there is an axis to compare to. I added an example with a 3rd variable also.